### ISSUE TITLE
More file/stream closing improvements

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8634]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8634|
|Type|enhancement|
|Created|2016-07-20 09:09:52Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8634 8634
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469005792489364 1469005792489364
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/363

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset dfc3aaaa83353bb5201089af3907a70b5b4ea1a6
```
#!CommitTicketReference repository="" revision="dfc3aaaa83353bb5201089af3907a70b5b4ea1a6"
Merge pull request #363 from scop/8634-scop-closing

Author: scop
Reviewer: rodrigc
Fixes: #8634

In a few tests, close some files that were left open.
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
